src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs

```diff
@@ -86,6 +92,11 @@ public ReadOnlyCollection<CameraDeviceInformation> GetDeviceInformation()
 
         internal static ReadOnlyCollection<CameraDeviceInformation> GetDeviceInformation(Native.CameraDeviceListStruct list)
         {
+            if (list.count == 0)
```

Comment:
IMO, I think empty collection is more safe and meaningful than null

Reply:
Right, I'll return Empty instead of null.